src/hrflow_connectors/__init__.py

```diff
@@ -1,11 +1,12 @@
+from hrflow_connectors.connectors.sapsuccessfactors.connector import SAPSuccessFactors
```

Comment:
Tu peux modifier ton fichier `__init__.py` en 
```py
from hrflow_connectors.connectors.sapsuccessfactors.connector import SAPSuccessFactors
```
Et modifié cette ligne afin d'avoir de l'homogénéité dans ton code 
```py
from hrflow_connectors.connectors.sapsuccessfactors import SAPSuccessFactors
```